arm-streamanalytics/2016-03-01/swagger/transformations.json

```diff
@@ -0,0 +1,312 @@
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "IfMatch",
+            "description": "The ETag of the transformation. Omit this value to always overwrite the current transformation. Specify the last-seen ETag value to prevent accidentally overwritting concurrent changes."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StreamingJobNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TransformationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The transformation was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/Transformation"
+            },
+            "headers": {
+              "ETag": {
+                "description": "The current entity tag for the transformation. This is an opaque string. You can use it to detect whether the resource has changed between requests. You can also use it in the If-Match or If-None-Match headers for write operations for optimistic concurrency.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Transformations"
+        ],
+        "operationId": "Transformations_Get",
+        "description": "Gets details about the specified transformation.",
+        "x-ms-examples": {
+          "Get a transformation": { "$ref": "../examples/Transformation_Get.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/StreamingJobNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TransformationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified transformation.",
+            "schema": {
+              "$ref": "#/definitions/Transformation"
+            },
+            "headers": {
+              "ETag": {
+                "description": "The current entity tag for the transformation. This is an opaque string. You can use it to detect whether the resource has changed between requests. You can also use it in the If-Match or If-None-Match headers for write operations for optimistic concurrency.",
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Transformation": {
```

Comment:
@veronicagg Correct. Transformation has no location property associated with it. This is also true for Inputs, Outputs, and Functions.